hpe_3par_kubernetes_manager.py

```diff
@@ -2456,6 +2556,18 @@ def get_pod_node(yml):
         logging.getLogger().error("Exception in get_pod_node :: %s" % e)
         raise e
 
+def get_current_node_of_pod(obj):
+    try:
+        node_name = None
+        logging.getLogger().info("\nReading current node name for pod %s " % obj.items[0].metadata.name)
+        command = "kubectl get pod %s -n %s -o jsonpath='{.spec.nodeName}'" % ( obj.items[0].metadata.name, obj.items[0].metadata.namespace)
+        node_name = get_command_output_string(command)

```

Comment:
Please ensure that you will receive the expected node_name.

Reply:
There is actually no expected node-name as it will differ based on which cluster it is run on, and which node it was on originally. This part can be checked in the testcase code as required.